spark/src/test/scala/org/apache/comet/exec/CometAggregateSuite.scala

```diff
@@ -45,7 +45,13 @@ class CometAggregateSuite extends CometTestBase with AdaptiveSparkPlanHelper {
       val filename = path.toString
       val random = new Random(42)
       withSQLConf(CometConf.COMET_ENABLED.key -> "false") {
-        ParquetGenerator.makeParquetFile(random, spark, filename, 10000, DataGenOptions())
+        ParquetGenerator.makeParquetFile(
+          random,
+          spark,
+          filename,
+          10000,
+          SchemaGenOptions(),
+          DataGenOptions())
```

Comment:
Is this intended to use the constructor for `DataGenOptions` or the default value in CometTestBase?

Reply:
This is using constructor for DataGenOptions, so using the default values.

The previous code was also doing this, but this PR added the separate `SchemaGenOptions()`.